clients.json

```diff
@@ -544,6 +544,16 @@
     "authors": ["vitaliykhamin"]
   },
 
+  {
+    "name": "laserdisc",
+    "language": "Scala",
+    "repository": "https://github.com/laserdisc-io/laserdisc",
+    "description": "Future free Fs2 native pure FP Redis client http://laserdisc.io",
+    "authors": ["sirocchj", "barambani"],
+    "recommended": true,
```

Comment:
This property should not be a part of the PR - it is left to the maintainers' discretion based on community feedback.

Reply:
comments addressed.